src/table/tableview/table-view.component.js

```diff
@@ -3,6 +3,7 @@ angular.module('patternfly.table').component('pfTableView', {
     config: '<?',
     dtOptions: '<?',
     colummns: '<',
+    colvis: '<',
```

Comment:
(1) colvis is a terminology of DataTables plugin. From my point of view, we should respect the original ideas of authorship and usage habit of veteran datatables users. FYI, https://datatables.net/reference/button/colvis.